src/sinks/hdfs/mod.rs

```diff
@@ -0,0 +1,75 @@
+//! `hdfs` sink.
+//!
+//! This sink will send it's output to hdfs.
```

Comment:
```suggestion
//! This sink will send it's output to HDFS.
```

**nit-picking (not blocking)**: this seems to be the proper casing for the name.